bin/git/hooks/post-merge-warn-env-changes.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# MIT © Sindre Sorhus - sindresorhus.com
+
+# git hook to run a command after `git pull` if a specified file was changed
+# Run `chmod +x post-merge` to make it executable then put it into `.git/hooks/`.
+
+changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
+
+check_run() {
+	echo "$changed_files" | grep --quiet "$1" && eval "$2"
+}
+
+warn_env_update() {
+  echo ""
+	echo "========================================================"
+	echo "================= ENV FILE UPDATED ====================="
```

Comment:
https://github.com/sjungwirth/githooks/blob/master/bin/git/hooks/pre-commit-reject-debug

original file I copied it from. no blinking, but I can throw in a badass skull if you want ;)